SCF-Session6-Question2/src/meta/utility/Polynomial.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * This class defines structure for polynomial 
+ * This contains immutable variable of polynomial
+ * @author ABHISHEK BISHNOI
+ * @version 1.0
+ * @since 22-July-2019
+ */
+package meta.utility;
+
+public class Polynomial {
+	/*
+	 * This Array Contains 2 variables in each row, these are as follows:
+	 * at index 0 - coficient value
+	 * at index 1 - power for that coficient
+	 * let's see 3y2+3y+3 is stored as
+	 * array[][] = { {3,2},{3,1},{3,0} }
+	 */
+	private final int polynomial[][]; //immutable variable
+	/*
+	 * This Constructor provides memory to int polynomial[]
+	 * @param int[][] 2-D Array
+	 * Complexity O(n)
+	 */
+	public Polynomial(int polynomial[][]){
```

Comment:
If we sort the polynomial data structure on the power, a lot more things would be easier ahead for all operations ?